components/producers/github-code-scanning/Makefile

```diff
@@ -12,6 +11,3 @@ container:
 					--file Dockerfile \
 					$$([ "${SOURCE_CODE_REPO}" != "" ] && echo "--label=org.opencontainers.image.source=${SOURCE_CODE_REPO}" ) \
 					 ../../../bin 1>&2
-
-publish:
-	$(DOCKER) push $(CONTAINER_REPO)/components/producers/github-code-scanning:$(DRACON_VERSION) 1>&2
```

Comment:
The Dockerfile doesn't seem to be doing anything other than wrapping the executable and adding certificates in it